stacks-coordinator/src/bitcoin_node.rs

```diff
@@ -150,7 +150,7 @@ impl LocalhostBitcoinNode {
         self.call_path(
             method,
             params,
-            Some(&format!("wallet/{}", self.wallet_name)),
+            Some(&format!("/wallet/{}", self.wallet_name)),
```

Comment:
Also, no need to change this if incorporating the update below
```suggestion
            Some(&format!("wallet/{}", self.wallet_name)),
```